core/tx_pool.go

```diff
@@ -173,6 +171,10 @@ func (config *TxPoolConfig) sanitize() TxPoolConfig {
 		log.Warn("Sanitizing invalid txpool price limit", "provided", conf.PriceLimit, "updated", DefaultTxPoolConfig.PriceLimit)
 		conf.PriceLimit = DefaultTxPoolConfig.PriceLimit
 	}
+	if isQuorum && (conf.PriceLimit == DefaultTxPoolConfig.PriceLimit) {
```

Comment:
This will allow Quorum users to have a price of zero, or > 1. However a value of 1 will be reset to zero... I don't think this is what you want to achieve.

Reply:
Yes. Another testcase was failing and as this DefaultTxPoolConfig was used in multiple files, I didn't want to impact all that code. 
Since then I have revised the code and will be resubmitting it with a zero setting instead of 1.

Reply:
Yes. Another testcase was failing and as this DefaultTxPoolConfig was used in multiple files, I didn't want to impact all that code.
Since then I have revised the code and will be resubmitting it with a zero setting instead of 1.